presto-docs/src/main/sphinx/release/release-0.189.rst

```diff
@@ -0,0 +1,81 @@
+* Fix client hang with ``LIMIT`` queries when ``optimizer.force-single-node-output``
+  is disabled.
+* Fix a bug in the :func:`bing_tile_at` and :func:`bing_tile_polygon` functions
+  that may produce incorrect results for points that are close to tile edges.
+* Fix scope semantics when lambda argument shadows relation column.
+* Improve error message when using catalog that does not exist.
+* Improve performance for queries with highly selective filters.
+* Execute :doc:`/sql/use` on the server rather than in the CLI, allowing it
+  to be supported by any client. This requires clients to add support for
+  the protocol changes (otherwise the statement will be silently ignored).
+* Allow casting JSON with missing fields to :ref:`row_type`.
+* Add support for dereference expressions to relation columns from lambda body.
+
+Security Changes
+----------------
+
+* Support configuring multiple authentication types, which allows supporting
+  clients that have different authentication requirements or gracefully
+  migrating between authentication types without needing to update all clients
+  at once. Specify multiple values for ``http-server.authentication.type``,
+  separated with commas.
+* Add support for TLS client certificates as an authentication mechanism by
+  specifying ``CERTIFICATE`` for ``http-server.authentication.type``.
+  The distinguished name from the validated certificate will be provided as a
+  ``javax.security.auth.x500.X500Principal``. The certificate authority (CA)
+  used to sign client certificates will be need to be added to the HTTP server
+  KeyStore (should technically be a TrustStore but separating them out is not
+  yet supported).
+* Skip sending final leg of SPNEGO authentication when using Kerberos.
+
+SPI Changes
+-----------
+
+* Deprecate ``SliceArrayBlock``.
+* Add SessionPropertyConfigurationManager plugin to enable overriding default
+  session properties dynamically.
+
+JDBC Driver Changes
+-------------------
+
+* Per the JDBC specification, close the ``ResultSet`` when ``Statement`` is closed.
+* Add support for TLS client certificate authentication by configuring the
+  ``SSLKeyStorePath`` and ``SSLKeyStorePassword`` parameters.
+* Add support for transactions using SQL statements or the standard JDBC mechanism.
+* Allow executing the ``USE`` statement. Note that this is primarily useful when
+  running arbitrary SQL on behalf of users. For programmatic use, continuing
+  to use ``setCatalog()`` and ``setSchema()`` on ``Connection`` is recommended.
+* Allow executing ``SET SESSION`` and ``RESET SESSION``.
+* Change mapping for columns with DECIMAL(p,s) data type. Previously such columns
+  were mapped to Presto's DOUBLE type and now to corresponding DECIMAL type.
+
+Hive Changes
+------------
+
+* Do not require setting ``hive.metastore.uri`` when using the file metastore.
+* Reduce memory usage when reading string columns from ORC or DWRF files.
+
+Thrift Changes
```

Comment:
I'd say "Thrift connector Changes" or "Thrift connector". Below you just write "Kafka connector".

Reply:
We capitalize words in headers. Adding “Conmector” is fine since Thrift is a bit ambiguous.

Just like we say “JDBC Driver” to disambiguate.

Maybe we should say connector on all of them to be consistent?